.changeset/quick-sheep-cross.md

```diff
@@ -0,0 +1,9 @@
+---
+"@finos/legend-art": minor
+"@finos/legend-extension-external-language-morphir": minor
+"@finos/legend-graph": minor
+"@finos/legend-studio": minor
+"@finos/legend-studio-app": minor
+---
+
+Add Legend Studio extension for external language Morphir.
```

Comment:
Please split this into multiple changeset, see https://github.com/finos/legend-studio/blob/master/CONTRIBUTING.md#changeset